server/helpers/aurhors-id-helper.js

```diff
@@ -0,0 +1,13 @@
+const getAuthorsIdFromArticle = (req, obj) => {
+  const allUserIds = obj.map(item => item.Article.author.id);
+
+  const uniqueIds = [...new Set(allUserIds)];
+
+  const removeUserOwnId = uniqueIds.filter(
```

Comment:
Give this is a better name. This sounds like the name of a function

Reply:
👍🏾